packages/devextreme/js/__internal/scheduler/header/m_date_navigator.ts

```diff
@@ -37,28 +40,36 @@ export const getDateNavigator = (header, item) => {
   };
 };
 
-const getPreviousButtonOptions = (header) => ({
-  key: 'previous',
-  icon: 'chevronprev',
-  elementAttr: { class: PREVIOUS_BUTTON_CLASS },
-  clickHandler: () => header._updateDateByDirection(DIRECTION_LEFT),
-  onContentReady: (e) => {
-    const previousButton = e.component;
-    previousButton.option('disabled', isPreviousButtonDisabled(header));
+const getPreviousButtonOptions = (header) => {
+  const viewAria = getViewAria(header.currentView.type);
+  const ariaMessage = messageLocalization.format(`dxScheduler-navigationPrevious-${viewAria}`);
```

Comment:
@Raushen @pomahtri  , please review the latest commit. I haven't considered the intervalCount because if I will create it, we may not use message localization due to language differences. If I'm not to use messageLocalization, then the aria-label will only be available in english..

Reply:
I am thinking of using pattern for message localization
`Next {0} day{1}`
but as @Raushen told me, it can't be this way since there are differences in language like Greek and Russian..